### PR TITLE
Added draft of unconf security packages recap

### PR DESCRIPTION
**Pull request by karthik**

### CODE REVIEW

**Comment by karthik** [comment]
@stefaniebutland My recap is in. Let me know if you need me to edit/clarify anything and what date to set this to. 🙏

File: content/blog/2018-06-06-2018-unconf-security-recap.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- ropensci
+- meetings
+- unconf
+---
+
+As part of our series summarizing all projects from this year's unconf I'm excited to dive into all the security related offerings from this year.
+
+# `middlechild`
+
+**Summary:** This package provides a R interface to the Man-in-the-middle (MITM) proxy and allows R users to intercept, modify, and introspect network traffic. The package provides functionality to download, install, configure and launch `mitmproxy`. In addition to helping test API packages and identify dangerous calls, R users can also route their browser traffic through a specific port and capture all network traffic in R.
```
Comment (stefaniebutland) [inline content/blog/2018-06-06-2018-unconf-security-recap.md]: "an" R interface?

File: content/blog/2018-06-06-2018-unconf-security-recap.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+
+Unconf projects x: 
+
+---
+slug: "unconf_recap_x"
+title: "Unconf18 projects x: middlechild, defender, ropsec, keybase"
+date: 2018-06-xx
```
Comment (stefaniebutland) [inline content/blog/2018-06-06-2018-unconf-security-recap.md]: date: 2018-06-06

File: content/blog/2018-06-06-2018-unconf-security-recap.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+
+Unconf projects x: 
+
+---
+slug: "unconf_recap_x"
```
Comment (stefaniebutland) [inline content/blog/2018-06-06-2018-unconf-security-recap.md]: slug: unconf_recap_2

File: content/blog/2018-06-06-2018-unconf-security-recap.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+date: 2018-06-xx
+authors:
+  - name: Karthik Ram
+categories: blog
+topicid: 
+tags:
+- community
+- ropensci
+- meetings
+- unconf
```
Comment (stefaniebutland) [inline content/blog/2018-06-06-2018-unconf-security-recap.md]: pls add tag
`- unconf18`

File: content/blog/2018-06-06-2018-unconf-security-recap.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+# `middlechild`
+
+**Summary:** This package provides a R interface to the Man-in-the-middle (MITM) proxy and allows R users to intercept, modify, and introspect network traffic. The package provides functionality to download, install, configure and launch `mitmproxy`. In addition to helping test API packages and identify dangerous calls, R users can also route their browser traffic through a specific port and capture all network traffic in R. 
+
+**Team:** [Ildiko Czeller](https://github.com/czeildi), [Karthik Ram](https://github.com/karthik), [Bob Rudis](https://github.com/hrbrmstr), [Kara Woo](https://github.com/karawoo)   
+
+**GitHub:** [https://github.com/ropenscilabs/middlechild](https://github.com/ropenscilabs/middlechild)
+
+# `defender`
```
Comment (stefaniebutland) [inline content/blog/2018-06-06-2018-unconf-security-recap.md]: for pkg name headings Maëlle used `###`. I haven't previewed yours - what heading type do you recommend that others can copy?

File: content/blog/2018-06-06-2018-unconf-security-recap.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+  - name: Karthik Ram
+categories: blog
+topicid: 
+tags:
+- community
+- ropensci
+- meetings
+- unconf
+---
+
```
Comment (stefaniebutland) [inline content/blog/2018-06-06-2018-unconf-security-recap.md]: Please add this where appropriate:

"In the spirit of exploration and experimentation at rOpenSci unconferences, these projects are not necessarily finished products or in scope for rOpenSci packages."

File: content/blog/2018-06-06-2018-unconf-security-recap.md (hunk 3 above)
Comment (stefaniebutland) [reply content/blog/2018-06-06-2018-unconf-security-recap.md]: @karthik this is recap 2 so

slug: "unconf_recap_2"
title: "Unconf18 projects 2: middlechild, defender, ropsec, keybase"

File: content/blog/2018-06-06-2018-unconf-security-recap.md (hunk 4 above)
Comment (stefaniebutland) [reply content/blog/2018-06-06-2018-unconf-security-recap.md]: pls keep both tags
`- unconf`
`- unconf18`

**Comment by maelle** [comment]
Why not add the cool defender&middlechild logos in the post? 😸

**Pull request merged**